### PR TITLE
Update Japanese translation(Line 2888-5094)

### PR DESCRIPTION
**Pull request by ziyi-xie**
This PR modifies some translations in the [former PR](https://github.com/kubernetes/dashboard/pull/7079) and updates the Japanese translation from line 2888 to line 5094. Please let me know if there are any mistakes. Thank you !
/hold for Japanese developer's review.
/assign @shu-mutou @atoato88

### CODE REVIEW

File: i18n/ja/messages.ja.xlf [hunk 1]
```diff
@@ -3266,175 +3266,175 @@
       </trans-unit>
       <trans-unit id="6887151649401555160" datatype="html">
         <source>SE Linux User</source>
-        <target state="new">SE Linux User</target>
+        <target>SE リナックスユーザー</target>
```
Comment (atoato88) [inline i18n/ja/messages.ja.xlf:3269]: I think the ja translation for `Linux` can be left as it is.
```suggestion
        <target>SE Linux ユーザー</target>
```

File: i18n/ja/messages.ja.xlf [hunk 2]
```diff
@@ -3266,175 +3266,175 @@
...
         <context-group purpose="location">
           <context context-type="sourcefile">src/app/frontend/common/components/securitycontext/template.html</context>
           <context context-type="linenumber">23</context>
         </context-group>
       </trans-unit>
       <trans-unit id="4972463622417704805" datatype="html">
         <source>SE Linux Role</source>
-        <target state="new">SE Linux Role</target>
+        <target>SE リナックスロール</target>
```
Comment (atoato88) [inline i18n/ja/messages.ja.xlf:3277]: ditto
```suggestion
        <target>SE Linux ロール</target>
```

File: i18n/ja/messages.ja.xlf [hunk 3]
```diff
@@ -3266,175 +3266,175 @@
...
         <context-group purpose="location">
           <context context-type="sourcefile">src/app/frontend/common/components/securitycontext/template.html</context>
           <context context-type="linenumber">28</context>
         </context-group>
       </trans-unit>
       <trans-unit id="7531425010556303342" datatype="html">
         <source>SE Linux Type</source>
-        <target state="new">SE Linux Type</target>
+        <target>SE リナックスタイプ</target>
```
Comment (atoato88) [inline i18n/ja/messages.ja.xlf:3285]: ditto
```suggestion
        <target>SE Linux タイプ</target>
```

File: i18n/ja/messages.ja.xlf [hunk 4]
```diff
@@ -3266,175 +3266,175 @@
...
         <context-group purpose="location">
           <context context-type="sourcefile">src/app/frontend/common/components/securitycontext/template.html</context>
           <context context-type="linenumber">33</context>
         </context-group>
       </trans-unit>
       <trans-unit id="5087843362946222470" datatype="html">
         <source>SE Linux Level</source>
-        <target state="new">SE Linux Level</target>
+        <target>SE リナックスレベルl</target>
```
Comment (atoato88) [inline i18n/ja/messages.ja.xlf:3293]: ditto
```suggestion
        <target>SE Linux レベルl</target>
```

File: i18n/ja/messages.ja.xlf [hunk 5]
```diff
@@ -3266,175 +3266,175 @@
...
         <context-group purpose="location">
           <context context-type="sourcefile">src/app/frontend/common/components/securitycontext/template.html</context>
           <context context-type="linenumber">38</context>
         </context-group>
       </trans-unit>
       <trans-unit id="3513903697113537912" datatype="html">
         <source>Windows GMSA Credential Spec Name</source>
-        <target state="new">Windows GMSA Credential Spec Name</target>
+        <target>ウィンドウズ GMSA クレデンシャル仕様名</target>
```
Comment (atoato88) [inline i18n/ja/messages.ja.xlf:3301]: It's a same with comment about `Linux` above.
```suggestion
        <target>Windows GMSA クレデンシャル仕様名</target>
```

File: i18n/ja/messages.ja.xlf [hunk 6]
```diff
@@ -3266,175 +3266,175 @@
...
         <context-group purpose="location">
           <context context-type="sourcefile">src/app/frontend/common/components/securitycontext/template.html</context>
           <context context-type="linenumber">44</context>
         </context-group>
       </trans-unit>
       <trans-unit id="7696069555285576911" datatype="html">
         <source>Windows GMSA Credential Spec</source>
-        <target state="new">Windows GMSA Credential Spec</target>
+        <target>ウィンドウズ GMSA クレデンシャル仕様</target>
```
Comment (atoato88) [inline i18n/ja/messages.ja.xlf:3309]: ditto
```suggestion
        <target>Windows GMSA クレデンシャル仕様</target>
```

File: i18n/ja/messages.ja.xlf [hunk 7]
```diff
@@ -3266,175 +3266,175 @@
...
         <context-group purpose="location">
           <context context-type="sourcefile">src/app/frontend/common/components/securitycontext/template.html</context>
           <context context-type="linenumber">49</context>
         </context-group>
       </trans-unit>
       <trans-unit id="6031271337180125854" datatype="html">
         <source>Windows Run as User</source>
-        <target state="new">Windows Run as User</target>
+        <target>ウィンドウズ実行ユーザー</target>
```
Comment (atoato88) [inline i18n/ja/messages.ja.xlf:3317]: ditto
```suggestion
        <target>Windows 実行ユーザー</target>
```

**Review by atoato88 — COMMENTED** [review]
I left some comments, please check these.

File: i18n/ja/messages.ja.xlf [hunk 8]
```diff
@@ -226,7 +226,7 @@
       </trans-unit>
       <trans-unit id="8433565378732329468" datatype="html">
         <source><x id="INTERPOLATION" equiv-text="&quot;\uFFFD0\uFFFD&quot;"/> <x id="INTERPOLATION_1" equiv-text="&quot;\uFFFD1\uFFFD&quot;"/> will be updated to reflect the desired replicas count.</source>
-        <target><x id="INTERPOLATION" equiv-text="&quot;\uFFFD0\uFFFD&quot;"/> <x id="INTERPOLATION_1" equiv-text="&quot;\uFFFD1\uFFFD&quot;"/> は、目標のレプリカ数になるように更新されます。</target>
+        <target><x id="INTERPOLATION" equiv-text="{{ data.displayName }}"/> <x id="INTERPOLATION_1" equiv-text="{{ data.objectMeta.name }}"/> は、目標のレプリカ数になるように更新されます。</target>
```
Comment (atoato88) [inline i18n/ja/messages.ja.xlf:229]: I don't know how to review the update which changes `"&quot;\uFFFD0\uFFFD&quot;"` to `"{{ data.displayName }}"`.

@shu-mutou 
Any comments?

Comment (ziyi-xie) [reply i18n/ja/messages.ja.xlf:229]: I checked the former translation [like this one](https://github.com/kubernetes/dashboard/blob/d2f2023824acd6eba6d1a6376be9b4da25d6efff/i18n/ja/messages.ja.xlf#L164 )  and noticed that all of the `&quot;\uFFFD0\uFFFD&quot;` have been changed to the value displayed in the following path, [like this](https://github.com/kubernetes/dashboard/blob/d2f2023824acd6eba6d1a6376be9b4da25d6efff/src/app/frontend/common/dialogs/download/template.html#L22), therefore I modified all of them using the same method. Could you confirm if my understanding is correct or not? Many thanks!

Comment (atoato88) [reply i18n/ja/messages.ja.xlf:229]: Thank you for the detailed explanation, It looks correct :smiley:

Comment (atoato88) [reply i18n/ja/messages.ja.xlf:229]: I've checked similar updates in this PR and it's OK.

File: i18n/ja/messages.ja.xlf [hunk 9]
```diff
@@ -3266,175 +3266,175 @@
...
+        <target>SE Linux タイプ</target>
         <context-group purpose="location">
           <context context-type="sourcefile">src/app/frontend/common/components/securitycontext/template.html</context>
           <context context-type="linenumber">33</context>
         </context-group>
       </trans-unit>
       <trans-unit id="5087843362946222470" datatype="html">
         <source>SE Linux Level</source>
-        <target state="new">SE Linux Level</target>
+        <target>SE Linux レベルl</target>
```
Comment (atoato88) [inline i18n/ja/messages.ja.xlf:3293]: Remove last `l`.
```suggestion
        <target>SE Linux レベル</target>
```

File: i18n/ja/messages.ja.xlf (hunk 8 above)
Comment (ziyi-xie) [reply i18n/ja/messages.ja.xlf:229]: Thank you so much for you confirmation!

**Review by shu-mutou — CHANGES_REQUESTED** [review]
Please fix _target_ tags

File: i18n/ja/messages.ja.xlf (hunk 8 above)
Comment (shu-mutou) [reply i18n/ja/messages.ja.xlf:229]: Content of equiv-text attribute does not seem to affect to view. These INTERPOLATIONs would be replaced correctly when the angular builds the dashboard.

Comment (ziyi-xie) [reply i18n/ja/messages.ja.xlf:229]: Well noted with many thanks!

**Pull request merged**